tests/serializers/tests.py

```diff
@@ -278,6 +278,12 @@ def test_deterministic_mapping_ordering(self):
             'second_category_pk': categories[1],
         })
 
+    def test_deserialize_force_insert(self):
+        """Tests that deserialized content can be saved with force_insert as a parameter."""
+        serial_str = serializers.serialize(self.serializer_name, Article.objects.all())
+        models = serializers.deserialize(self.serializer_name, serial_str)
+        [model.save(force_insert=False) for model in models]
```

Comment:
Shouldn't this test fail if you're `force_insert`ing an object that already exists?